ament_cmake_python/cmake/ament_python_install_package.cmake

```diff
@@ -156,28 +160,18 @@ setup(
     DESTINATION "${ARG_DESTINATION}/${egg_install_name}.egg-info"
   )
 
-  if(ARG_SCRIPTS_DESTINATION)
-    file(MAKE_DIRECTORY "${build_dir}/scripts")  # setup.py may or may not create it
-
-    add_custom_target(
-      ament_cmake_python_build_${package_name}_scripts ALL
-      COMMAND ${python_interpreter} setup.py install_scripts -d scripts
-      WORKING_DIRECTORY "${build_dir}"
-      DEPENDS ${egg_dependencies}
-    )
-
-    if(NOT AMENT_CMAKE_SYMLINK_INSTALL)
-      # Not needed for nor supported by symlink installs
-      set(_extra_install_args USE_SOURCE_PERMISSIONS)
-    endif()
-
-    install(
-      DIRECTORY "${build_dir}/scripts/"
-      DESTINATION "${ARG_SCRIPTS_DESTINATION}/"
-      ${_extra_install_args}
-    )
+  if(POLICY CMP0087)
+    # Allow generator expressions in install(CODE...)
+    cmake_policy(SET CMP0087 NEW)
```

Comment:
@rhaschke which generator expressions?

Reply:
We don't use generator expressions, but cmake complains about the unset [CMP](https://cmake.org/cmake/help/git-stage/policy/CMP0087.html) otherwise.

Reply:
The warning is even reported by CI: https://ci.ros2.org/job/ci_linux/16094/cmake/NORMAL

Reply:
Hmm, I wonder if a generator expression is introduced by `build_dir`. Anyways, we need to cap that warning.

Reply:
> How do you trigger Full CI on this PR?

Using our [`ci_launcher`](https://ci.ros2.org/job/ci_launcher/) with a custom `ros2.repos` file (see https://ci.ros2.org/job/ci_launcher/9781). 

> Yes, that's what I did. 

What's funny is that I run CI yesterday and there have not been changes to this PR in weeks. Maybe https://github.com/ament/ament_cmake/pull/372/commits/fe6b3a38ad1391c13baca24912867753b2210e30 was not enough?